package.json

```diff
@@ -4,20 +4,11 @@
   "description": "Simple editor to create responsive content",
   "main": "./editor.js",
   "dependencies": {
-    "browserify": "^9.0.3",
```

Comment:
Well, I treat `dependencies` as a list of deps required in actual source code - production flow, leaving `devDependencies` for those used in build/test/... flows.

Of course, browserify merges all stuff together into one single prod file, but having all the deps listed under `devDependencies` omits a clear view at things required in different flows.